packages/atti-components/src/components/Stepper/styles/StyledStep.js

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { size } from 'lodash'
+
+import { useStepperDispatch, useStepper } from '../hooks'
+
+const StyledStep = styled.div`
+  position: relative;
+  display: inline-block;
+
+  width: ${({ theme }) => theme.components.stepper.sizes.element};
+  height: ${({ theme }) => theme.components.stepper.sizes.element};
+
+  border-radius: ${({ theme }) => theme.components.stepper.sizes.borderRadius};
+  cursor: pointer;
+  box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.2);
```

Comment:
We have a box-shadow in the theme. Maybe you can use it ?

Reply:
Yeah, I'm gonna use it.